lib/drab.ex

```diff
@@ -294,6 +312,19 @@ defmodule Drab do
     socket
   end
 
+  # Handles callbacks synchronously
+  # Temporary disabled until it will be needed, for avoiding warning
+  # @spec handle_callback_sync(Phoenix.Socket.t(), atom, atom) :: Phoenix.Socket.t()
+  # defp handle_callback_sync(socket, commander, callback) do
+  #   if callback, do: apply(commander, callback, [socket])
+  #   socket
+  # end
+
+  @spec handle_callback_sync(atom, atom) :: any
+  defp handle_callback_sync(commander, callback) do
+    if callback, do: apply(commander, callback, [])
```

Comment:
This really shouldn't exist, like really really shouldn't exist.

If someone wants the pid to drab they can just call the `Drab.pid(socket)` as normal in the `onload` callback (or any other callback).  The presence of this function makes it easy to accidentally perform complex operations and implies they can only get the pid via this function instead of just using `Drab.pid(socket)` as normal.

Reply:
This is necessary to let to subscribe to PubSub in the Commander. This because it is deprecated to pass a PID to the PubSub.subscribe function, otherwise we could just use unload event and pass the Drab.pid(socket) to PubSub. This is the reason behind the choice to have a new sync handler. I don't really like it either, but afaik there is no other ways to implement a pubsub subscription from inside a Commander.